Automattic/jetpack
# Connection: send package versions on init hook

Fixes #21165

#### Changes proposed in this Pull Request:
* Send the package versions to wpcom on the `init` hook instead of `plugins_loaded `to prevent a conflict with a third party plugin. The plugin isn't ready until the `after_setup_theme` action has fired.

* Is there any reason that we shouldn't move the package version check and wpcom request to the init hook? This seems like a safe change to me.

#### Jetpack product discussion
* n/a

#### Does this pull request change what data or activity we track or use?
* no

#### Testing instructions:
1. On a brand new site, install the Passwords Evolved plugin.
2. Then, install, activate, and connect Jetpack version 10.0.
3. Then, change from 10.0 to this branch.
4. Verify that no fatal errors occur.
5. Verify that the `jetpack_package_versions` option has been set: `wp option get jetpack_package_versions`
